cmd/package-operator-manager/bootstrap/bootstrap_test.go

```diff
@@ -94,7 +96,12 @@ func TestBootstrapper_bootstrap(t *testing.T) {
 	t.Parallel()
 
 	c := testutil.NewClient()
-	b := &Bootstrapper{client: c}
+	// Create a test logger and add it to the bootstrapper
+	logger := testr.New(t)
+	b := &Bootstrapper{
+		client: c,
+		log:    logger,
```

Comment:
What do you think about dropping the field Bootstrapper.log completely because the context that is passed to Bootstrapper.bootstrap already contains a logger?